docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,89 @@
+How to configure boot parameters on Ubuntu Core 
+================================================
+
+.. note::
+
+    This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
+
+Ubuntu Core doesn't have the :code:`/etc/default/grub` file mapped for the kernel parameters, nor the `update-grub`_ utility.
+On Ubuntu Core, the kernel parameters can be modified statically or dynamically.
+Static configurations are added during the image build.
+On the other hand, dynamic configurations are set `using snap options`_ on a running system.
+This guide focuses on the dynamic configuration method, suitable for iteratively tuning the operating system.
+
+.. hint::
+    
+    Refer to :doc:`../how-to/uc-image-creation` if you want to create a real-time Ubuntu Core image.
+
+Open a terminal and access your real-time Ubuntu Core instance.
+The instance should be on a physical device rather than a virtual machine.
+While a virtual machine can be used to follow the steps in this guide, it will not provide the benefits of a real-time kernel.
+
+There are two snap options for setting the kernel parameters:
+
+- `system.kernel.cmdline-append`_
+- `system.kernel.dangerous-cmdline-append`_
+
+The first one is used to append permitted kernel boot parameters that are verified against an `allow list`_ in the `gadget snap`_. 
+The second one is used to append any other kernel boot parameters, considered as not analyzed by the gadget snap.
+
+When using the reference gadget snaps (`pc-gadget`_ and `pi-gadget`_), you need to use the `system.kernel.dangerous-cmdline-append` option, as the allow list is not present in them.
+
+When creating a custom `gadget snap`_ for a real-time Ubuntu Core system, it is advisable to add an allow list with the relevant parameters.
+
+The kernel parameters in the snap configurations exactly match the `kernel parameters`_ that would be passed to the kernel on the grub command line, the :code:`GRUB_CMDLINE_LINUX_DEFAULT` on :code:`/etc/default/grub` file. 
+Parameters are passed as a single string in the :code:`key=value` format, with each parameter separated by a space.
+
+In the case of :doc:`real-time kernel parameters <../reference/kernel-boot-parameters>`, the most interesting parameters are `nohz`_, `nohz_full`_ and `irqaffinity`_.
+The :code:`nohz` is used to enable/disable dynamic ticks (possible values are `on` or `off`). 
+The :code:`nohz_full` receives a `cpu list`_ specifying which CPUs will have dynamic ticks disabled. 
+The `irqaffinity` is used to set the affinity of the interrupt requests (IRQs) to the CPUs, in other words: which CPUs will handle the IRQs.
+
+For example:
```

Comment:
I know this is not a tutorial but it might be useful to help the user confirm the command is doing what they think it's doing with a brief summary.

```
For example, the snap command below enables dynamic ticks for at least 2 CPUs...
```

Reply:
@locnnil do you agree with https://github.com/canonical/real-time-ubuntu-docs/pull/45/commits/6db6a330989c8a7b9bd775476e79c26ea3876519 ?

Reply:
There are some problems with the statement on https://github.com/canonical/real-time-ubuntu-docs/commit/6db6a330989c8a7b9bd775476e79c26ea3876519:

- "enables dynamic ticks" : That's true, but you may want to add a URL on that, or just add on parenthesis: 
"(avoid sending scheduling-clock interrupts to CPUs with a single runnable task)" This is also referred to as `adaptive-ticks` in other places on the kernel, both have the same meaning, I would prefer to use `adaptive-ticks`, but I leave it up to you. You can check [this kernel doc snippet](https://github.com/torvalds/linux/blob/3ec3f5fc4a91e389ea56b111a73d97ffc94f19c6/Documentation/timers/no_hz.rst?plain=1#L112C1-L117C43) with a precise explanation on what NO_HZ is about.

- "disabling them for all CPUs 2-N" : This is wrong, it's not disabling `dynamic ticks`/`adaptive-ticks`, it is enabling it for the selected CPUs, [as you can check here](https://github.com/torvalds/linux/blob/3ec3f5fc4a91e389ea56b111a73d97ffc94f19c6/Documentation/timers/no_hz.rst?plain=1#L124-L125).

The final part of the sentence is corret.

Reply:
@locnnil 
> "disabling them for all CPUs 2-N" : This is wrong, it's not disabling dynamic ticks/adaptive-ticks, it is enabling it for the selected CPUs, [as you can check here](https://github.com/torvalds/linux/blob/3ec3f5fc4a91e389ea56b111a73d97ffc94f19c6/Documentation/timers/no_hz.rst?plain=1#L124-L125).

So the explanation above it is wrong, coming from #39.